docs/content/users/extend/database-types.md

```diff
@@ -2,7 +2,14 @@
 
 DDEV supports many versions of the MariaDB, MySQL, and PostgreSQL database servers.
 
-The default database type is MariaDB, and the default version is currently 10.4, but you can use MariaDB versions 5.5-10.8 and 10.11, MySQL 5.5-8.0, and Postgres 9-16. (New LTS versions of each of these are typically added soon after release. The very old versions are kept for compatibility with older projects.)
+The following database types are supported out of the box:
+
+- MariaDB 5.5-10.8 and 10.11
+- MySQL 5.5-8.0
+- Postgres 9-16
+
+The default database type is MariaDB, and the default version is currently 10.4.
+(New LTS versions of each of these are typically added soon after release. The very old versions are kept for compatibility with older projects.)
```

Comment:
```suggestion
New LTS versions of each of these are typically added soon after release. The very old versions are kept for compatibility with older projects.
```
I think it's better without parentheses when laid out like this.

Reply:
Agreed. I had it without, then worried about what Matt would say. :)